style.css

```diff
@@ -0,0 +1,323 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  font-family: "poppins", sans-serif;
+  background: #f7f7f9;
+  margin: 0;
+  height: 100vh;
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  background-image: url("./images/header-shapes-mobile.png"), url("./images/header-background-mobile.png");
+  background-repeat: no-repeat, no-repeat;
+  padding: 80px 25px 114px 25px;
+  background-color: #fff;
+  height: 70%;
+  border-bottom-left-radius: 5em;
+}
+
+.connection {
+  font-family: "poppins", sans-serif;
+  color: #7f8cff;
+  font-weight: 400;
+  font-size: 15px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+  width: 100%;
+}
+
+.buto {
+  border: none;
+  background-color: #fff;
+  padding: 23px 23px 10px 10px;
+}
+
+.logo {
+  font-family: "poppins", sans-serif;
+  color: #6070ff;
+  font-weight: bold;
+  font-size: large;
+  padding: 20px 10px 10px 25px;
+  text-decoration: none;
+}
+
+.intro {
+  font-family: "poppins", sans-serif;
+  color: #172b4d;
+  width: 60%;
+  font-weight: 900;
+  font-size: 18px;
+}
+
+.introduction {
+  font-family: "poppins", sans-serif;
+  color: #344563;
+  font-weight: 400;
+  font-size: 16px;
+  width: 100%;
+  margin-top: 0;
+}
+
+.social ul {
+  display: flex;
+  gap: 15px;
+  list-style: none;
+  padding: 0;
+}
+
+.scroll {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
+
+.box-contain {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #7a869a;
+  width: 90%;
+  margin: 0 auto;
+  border-radius: 16px;
+  background-color: #fff;
+  list-style: none;
+  margin-bottom: 50px;
+}
+
+.box-contain ul {
+  padding: 0;
+  margin: 0;
+}
+
+.ptflio-item {
+  width: 90%;
+  margin: 0 auto;
+  list-style: none;
```

Comment:
- [ ] **As requested by the previous reviewer**, Please kindly make sure your project is responsive on mobile screens. But your project has a horizontal scroll on the `375px` screen size. Think you should just the styling of your cards and containers to make them responsive to remove the horizontal scroll. 
  **You can see the screenshot below**. :pray: 
  ![Screenshot from 2022-12-03 21-41-17](https://user-images.githubusercontent.com/92303891/205463241-7263e022-1d9a-4aed-aa66-3fae4cf517d2.png)

  **You can see from this too**

  [Screencast from 12-03-2022 09:34:12 PM.webm](https://user-images.githubusercontent.com/92303891/205463225-c6b4b23c-7e55-47d3-a06f-48b3fd07502c.webm)